pkg/controller/controller.go

```diff
@@ -63,37 +68,11 @@ func NewController(shareRelist time.Duration) (*Controller, error) {
 	// informers.NewSharedInformerFactoryWithOptions, but we restrict OpenShift
 	// "system" namespaces with chatty configmaps like the leaderelection related ones
 	// that are updated every few seconds
-	//TODO make this list externally configurable
 	tweakListOptions := internalinterfaces.TweakListOptionsFunc(func(options *metav1.ListOptions) {
 		fsString := ""
 		namespaceFieldSelector := "metadata.namespace!=%s"
-		namespaces := []string{"kube-system",
-			"openshift-machine-api",
-			"openshift-kube-apiserver",
-			"openshift-kube-apiserver-operator",
-			"openshift-kube-scheduler",
-			"openshift-kube-controller-manager",
-			"openshift-kube-controller-manager-operator",
-			"openshift-kube-scheduler-operator",
-			"openshift-console-operator",
-			"openshift-controller-manager",
-			"openshift-controller-manager-operator",
-			"openshift-cloud-credential-operator",
-			"openshift-authentication-operator",
-			"openshift-service-ca",
-			"openshift-kube-storage-version-migrator-operator",
-			"openshift-config-operator",
-			"openshift-etcd-operator",
-			"openshift-apiserver-operator",
-			"openshift-cluster-csi-drivers",
-			"openshift-cluster-storage-operator",
-			"openshift-cluster-version",
-			"openshift-image-registry",
-			"openshift-machine-config-operator",
-			"openshift-sdn",
-			"openshift-service-ca-operator",
-		}
-		for _, ns := range namespaces {
+		for _, ns := range ignoredNamespaces {
+			klog.V(4).Infof("namespace '%s' is being ignored", ns)
 			nsfs := fmt.Sprintf(namespaceFieldSelector, ns)
 			if len(fsString) == 0 {
 				fsString = nsfs
```

Comment:
```suggestion
		for _, ns := range ignoredNamespaces {
			klog.V(4).Infof("namespace '%s' is being ignored", ns)
			options.FiledSelector = fmt.Sprintf(namespaceFieldSelector, ns)
		}
```
Is there something I am missing, or can this all be refactored to the above suggestion?